tests/template_tests/syntax_tests/test_exceptions.py

```diff
@@ -20,12 +20,10 @@ def test_exception02(self):
         """
         Raise exception for invalid variable template name
         """
-        if self.engine.string_if_invalid:
-            with self.assertRaises(TemplateDoesNotExist):
-                self.engine.render_to_string('exception02')
-        else:
-            with self.assertRaises(TemplateSyntaxError):
-                self.engine.render_to_string('exception02')
+        with self.assertRaises(TemplateSyntaxError) as exception_ctx:
```

Comment:
You can use `assertRaisesMessage` here rather than checking the message separately.